{
    "exception": "ValueError",
    "exception_text": "Does not validate "
}
